deploy/kubernetes/eck/es_kibana.yaml

```diff
@@ -0,0 +1,40 @@
+# This sample sets up a an Elasticsearch cluster along with a Kibana instance,
+# configured to be able to communicate with each other
+apiVersion: elasticsearch.k8s.elastic.co/v1alpha1
```

Comment:
We have recently released ECK 1.0.0-beta1 which changes the API version and some of the attributes. We tried to document here https://www.elastic.co/guide/en/cloud-on-k8s/1.0/k8s-upgrading-eck.html#k8s-convert-manifests what changed.